__tests__/app.test.js

```diff
@@ -27,15 +27,59 @@ describe("/api", () => {
           });
         });
     });
-    test("404: returns an error message when passed an invalid url path", () => {
-      return request(app)
-        .get("/api/categos")
-        .expect(404)
-        .then((response) => {
-          expect(response.body).toEqual({
-            message: "404: invalid end point provided",
+  });
+  describe("/api/reviews", () => {
+    describe("/api/reviews/:review_id", () => {
+      test("200: responds with a review object matching the passed id", () => {
+        return request(app)
+          .get("/api/reviews/1")
+          .expect(200)
+          .then((response) => {
+            const reviews = response.body.reviews;
+            expect(reviews.length === 1).toBe(true);
+            reviews.forEach((review) => {
+              expect(review.hasOwnProperty("review_id")).toBe(true);
+              expect(review.hasOwnProperty("title")).toBe(true);
+              expect(review.hasOwnProperty("review_body")).toBe(true);
+              expect(review.hasOwnProperty("designer")).toBe(true);
+              expect(review.hasOwnProperty("review_img_url")).toBe(true);
+              expect(review.hasOwnProperty("votes")).toBe(true);
+              expect(review.hasOwnProperty("category")).toBe(true);
+              expect(review.hasOwnProperty("owner")).toBe(true);
+              expect(review.hasOwnProperty("created_at")).toBe(true);
+            });
           });
-        });
+      });
```

Comment:
Your test says you're expecting a review object, but you get an array with an object in it. I would prefer that body.review is just a single object with the correct review on it. Furthermore id like some more specific testing that when i request review_id of 1, that i am indeed getting all the info associated with review id 1 and not just any review object with these keys on it. There is no assurance here that im getting the RIGHT object, just an object with these keys at the moment. I would look in the test data to find the correct data for the first review in the file